### DISCUSSION TITLE
Feature Request: Non Tailwind /bootstrap version

### DISCUSSION THREAD

**Discussion by uqjohart** [Power-Components/livewire-powergrid#865] [Ideas]
# ⚡ PowerGrid ⚡ Feature Request

## Summary

Love the library, but it would be next level amazing to have a non tailwind or bootstrap version for those of us who rather write out own themes without a ton of !important. 

### Why is this needed?

Would save us having to cannibalism the theme files and basically version lock ourselves out of updates. 
And it is always a really good idea to have a style system agnostic version available. 

### Suggested Solution(s)

1. Strip out the inline styles
2. Add a 'blank install' option on the install with a message saying "Hey this will look broken, you need to style it.
3. Add in some very sparingly contextual classes e.g. .filters, .data (but very very sparingly)

I know it is a chunk of work, but it would make it much more flexible for project that don;t want to be tied down. No hate on tailwind, totally makes sense on some projects. Others, it is good to be able to do it yourself. 

Again, love your work I can tell you sunk a ton of time into this.

**Comment by luanfreitasdev** [comment]
Hello!,

This is interesting, but it would need to change several things in the current structure because each theme (tailwind or bootstrap) has its structures that must be ready.
